### ISSUE TITLE
For CrossReplicaBN, may take the convenient from NVIDIA/apex which support sync BatchNorm

### ISSUE THREAD

**Issue by ksnzh** [AaronLeong/BigGAN-pytorch#18]

**Comment by ksnzh** [comment]
https://github.com/NVIDIA/apex
